sdk/eventhub/Azure.Messaging.EventHubs/tests/Compatibility/TrackOneSharedAccessTokenTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Messaging.EventHubs.Authorization;
+using Azure.Messaging.EventHubs.Compatibility;
+using NUnit.Framework;
+using TrackOne;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="TrackOneSharedAccessSignatureToken" />
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
+    [Category(TestCategory.BuildVerification)]
```

Comment:
How is `TestCategory.BuildVerification` different from just a normal test?

Reply:
It's an explicit statement of intent to make authors aware that there should be no tests with external dependencies appearing in the file.   Like other categories, it's simply metadata in the interest of being explicit and allowing filtering of test runs.

Reply:
Why invent a new thing when we already have a convention that tests without category should require no dependency?

Reply:
Why does adding a category to tests imply inventing a new thing?  I'd also question whether that convention is implied because the core team just happens to do it or is documented in a contributor's guide.   I'm not sure that I would agree that something on this small a scale would detract from uniformity or readability in a way that would warrant a guideline saying "here are the only categories for tests that can be used within the respository."

Reply:
> Why does adding a category to tests imply inventing a new thing? 

How isn't this a new thing?

You've created a TestCategory class internal to EventHubs that I thought we've agreed to avoid having (https://github.com/Azure/azure-sdk-for-net/pull/6430#discussion_r288694548) and use it to mark tests, to every new developer that starts looking at these tests this is a new concept to learn, different from what the rest of the repo is doing.

Reply:
`Category` exists and is a well-known construct in the NUnit library.    A string was set to create a piece of metadata.  I'd say that that differs from the implementation that was avoided in the short term, which wrapped things in a new attribute and used a custom enumeration.   I'd concede the point that a new attibute/enum was a new concept that held a small cognitive burden.  I would expect that a developer looking at a test suite would recognize "this is a category for the tests" with no additional burden.   

Respectfully, I find it useful to categorize and tests to aid in organization to prevent the mixing and matching of "these tests use live services, these don't" in the same class, which has a higher connotative burden for developers.   Would you have the same concern if the differentiation was strictly by filename `BlahUnitTests` or `BlahBvts` versus `BlahLiveTests` ... or encapsulated in a namespace?

Reply:
> I would expect that a developer looking at a test suite would recognize "this is a category for the tests" with no additional burden.

But then they need to figure out what `BuildVerification` is and how it differs from not having a category and the answer would be it doesn't. 

The only major test kind we have is live tests which we do mark with an attribute that we also filter on so it has organizational and practical usage. `BuildVerification` is just a complete opposite of Live that we already have.

I don't want to waste your time on minor details like these but the easiest way to avoid it is just not introducing new concepts that are not absolutely required.

I'm OK with file names, at least those are used more widely across the repo.

Reply:
Truthfully, I just want to be able to organize things in a way that makes it clear what does what and without having the need to drive consensus across a wide array of people with strong opinions and focused on different things.   If you'll accept file names without concern, I'll adapt to that.

Reply:
Done.   Live tests will make use of a file suffix to identify them.